models/stg_pendo.yml

```diff
@@ -98,9 +98,11 @@ models:
       - name: user_agent
         description: String identifying the browser and operating system on which the event was triggered.
       - name: visitor_id
-        description: Foreign key referencing the `VISITOR` who triggered the event. 
+        description: Foreign key referencing the `VISITOR` who triggered the event.
+      - name: _fivetran_id
```

Comment:
should we add _fivetran_id to the src_pendo.yml file as well?

Reply:
will need to regen docs after doing so

Reply:
Yes I agree we should do this. I have added for the new tables and also for the tables where _fivetran_id was added in v0.4.0.

Reply:
Yes I agree we should do this. I have added for the new tables and also for the tables where _fivetran_id was added in v0.4.0. I also went ahead and added it to the seeds.